Description: Fix cmake libusb search and fix libusb include
 Adjust cmake test for libusb library so that the compatible libusb for
 freebsd is found.  Since freebsd's libusb header are located differently
 than libusb-1.0, the includes in the source code must be adjusted using
 #include <libusb.h>.  Actually those new include statements are those that
 libusb-1.0 really expects to be used (see libusb-1.0.pc cflags to verify
 this).
Origin: vendor
Forwarded: https://github.com/OpenKinect/libfreenect/pull/265
Author: nicolas bourdaud <nicolas.bourdaud@gmail.com>
Last-Update: 2012-01-19

--- a/cmake_modules/Findlibusb-1.0.cmake
+++ b/cmake_modules/Findlibusb-1.0.cmake
@@ -49,7 +49,7 @@
 else (LIBUSB_1_LIBRARIES AND LIBUSB_1_INCLUDE_DIRS)
   find_path(LIBUSB_1_INCLUDE_DIR
     NAMES
-	libusb-1.0/libusb.h
+	libusb.h
     PATHS
       /usr/include
       /usr/local/include
@@ -61,7 +61,7 @@
 
   find_library(LIBUSB_1_LIBRARY
     NAMES
-      usb-1.0
+      usb-1.0 usb
     PATHS
       /usr/lib
       /usr/local/lib
@@ -95,4 +95,4 @@
   # show the LIBUSB_1_INCLUDE_DIRS and LIBUSB_1_LIBRARIES variables only in the advanced view
   mark_as_advanced(LIBUSB_1_INCLUDE_DIRS LIBUSB_1_LIBRARIES)
 
-endif (LIBUSB_1_LIBRARIES AND LIBUSB_1_INCLUDE_DIRS)
\ No newline at end of file
+endif (LIBUSB_1_LIBRARIES AND LIBUSB_1_INCLUDE_DIRS)
--- a/src/usb_libusb10.c
+++ b/src/usb_libusb10.c
@@ -28,7 +28,7 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #include <string.h>
-#include <libusb-1.0/libusb.h>
+#include <libusb.h>
 #include "freenect_internal.h"
 
 int fnusb_num_devices(fnusb_ctx *ctx)
--- a/src/usb_libusb10.h
+++ b/src/usb_libusb10.h
@@ -27,7 +27,7 @@
 #ifndef USB_LIBUSB10
 #define USB_LIBUSB10
 
-#include <libusb-1.0/libusb.h>
+#include <libusb.h>
 
 #if defined(__APPLE__)
 /*
